vuepress/docs/v7.1/docs/getting-started/ent-bundle.md

```diff
@@ -117,6 +119,15 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
     -  Maximum number of subtasks running simultaneously. The default value is 3. Setting `max-parallel=1` results in a sequential process.
 - `--stdout`:
     -  Print the subtask logs to the standard output (stdout) instead of individual log files. This is very useful in CI/CD pipelines.
+- `ent bundle pack -s` and `ent bundle images`:
+
+
+   - The `skipDockerBuild` flag enables custom image builds by preventing the `pack` command from creating Docker images for the bundle and microservices. Instead, the YAML descriptors and Dockerfile will be saved to the `.output` folder. After running `ent bundle images` to return the Docker image names and tags, the user can build the bundle image from the `.output` folder and each microservice image from its respective folder. 
+
+   OR
+
+   - Setting the `skipDockerBuild` option allows the user to create a custom image build. No images are generated for the bundle and microservices, but the YAML descriptors and Dockerfile are created in the `.output` folder. 
+   - The command `ent bundle images` returns the Docker image names and tags, which are used to manually build the bundle and microservice images in the `.output` and microservice folders, respectively.
```

Comment:
I like this much better. The first one, you have to decipher what is what. 

It should match the other flag formats on the page, like this one, it's much easier to read
`--fail-fast`:
    -  Fail the command as soon as any subtask fails
- `--max-parallel=<value>`:
    -  Maximum number of subtasks running simultaneously. The default value is 3. Setting `max-parallel=1` results in a sequential process.

Reply:
i liked the second better too. and yes, these bullets will fall under the commands listed above the first option